tests/integration/test_spansv2.py

```diff
@@ -1039,55 +1063,32 @@ def test_spansv2_attribute_normalization(
 
     relay.send_envelope(project_id, envelope)
 
-    assert spans_consumer.get_span() == {
-        "trace_id": "5b8efff798038103d269b633813fc60c",
-        "span_id": "eee19b7ec3c1b175",
-        "attributes": {
```

Comment:
I'd prefer if we kept asserting the full snapshot, it's a good regression test for other changes as well (e.g. maybe someone adds a new attribute to the http normalization).

To reduce the boilerplate, you can just assert the attributes and add in the common attributes vai `**common`.